src/Enterspeed.Source.UmbracoCms/Handlers/EnterspeedJobsHandler.cs

```diff
@@ -78,6 +79,22 @@ public void HandleJobs(IList<EnterspeedJob> jobs)
                 }
             }
 
+            // Remove existing failed jobs that was handled
+            RemoveFromFailedJobs(failedJobsToDelete);
+
+            // Save existing and new failed jobs
+            SaveFailedJobs(failedJobs);
```

Comment:
I guess we can move this into the if statement below and only save failed jobs if we have any